src/darker/git.py

```diff
@@ -110,6 +110,16 @@ def git_is_repository(path: Path) -> bool:
         return False
 
 
+def git_get_repo_root(cwd: Path) -> Path:
+    """Return the root directory of the Git repository containing the given path
+
+    :param cwd: The path to start the search from
+    :return: The root directory of the Git repository, or ``None`` if not found
```

Comment:
>```
>``None`` if not found
>```
doesn't seem to match what the code does.

It would be good to have unit tests for this code, including a scenario in which no root of a Git repository is found.

Reply:
Yep. I will add unit tests for this. And also one that reproduces the issue this pr hopes to fix.

Reply:
I've removed this code in favour of the black compat call.